src/storage/index/base_hash_index.cpp

```diff
@@ -0,0 +1,97 @@
+#include "storage/index/base_hash_index.h"
+
+#include "common/types/int128_t.h"
+#include "common/types/ku_string.h"
+#include "transaction/transaction.h"
+
+using namespace kuzu::common;
+using namespace kuzu::transaction;
+
+namespace kuzu::storage {
```

Comment:
Let's not use nested namespace here for now. We should change to nested namespace all for once if we decide to move to this style later, which I don't find it's important to do.